cps/main.c

```diff
@@ -85,12 +85,9 @@ send_arp_reply_kni(struct cps_config *cps_conf, struct cps_arp_req *arp)
 	struct rte_arp_hdr *arp_hdr;
 	size_t pkt_size;
 	struct rte_kni *kni;
-	struct rte_mempool *mp;
 	int ret;
 
-	mp = cps_conf->net->gatekeeper_pktmbuf_pool[
-		rte_lcore_to_socket_id(cps_conf->lcore_id)];
-	created_pkt = rte_pktmbuf_alloc(mp);
+	created_pkt = rte_pktmbuf_alloc(cps_conf->mp);
```

Comment:
As in the example above, the blocks CPS, GT, and LLS are creating packets that are not accounted for in their mbuf pools. Could you identify a reasonable value to add to their pools?